### ISSUE TITLE
Icons (SVGs) should have role="img" in them for proper semantics and best possible accessibility name

### ISSUE THREAD

**Issue by BogdanCerovac** [ramiroaisen/svelte-material-icons#20]
SVGs have no implicit aria roles on their own, so we must be able to set the role manually (with smart defaults).

As svelte-material-icons only includes icons (and not interactive SVGs etc.) a smart default should be role="img".

This would also enable developers to use best possible accessibility name pattern when icons are used inside interactive elements and interactive elements get their names based on SVG;

`<svg ... role="img"><title>{accessibility name}</title><!-- rest of svg --></svg>` as documented in [Carry Fishers Accessible SVG patterns comparison on Smashing Magazine](https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/#best-in-show-1)

I suggest therefore to introduce new prop to override default `role="img"` (or if we are 100% sure that icons will be only used as icons we can perhaps just hardcode `role="img"` for all SVGs...).

**Comment by jrasm91** [comment]
It looks like pattern 5 also works well, which is the one I think we will want to go with here.

**Comment by BogdanCerovac** [comment]
@jrasm91  - yes, `<svg ... role="img"><title>{accessibility name}</title><!-- rest of svg --></svg>` is actually pattern 5 and I suggest using it when possible.

**Comment by jrasm91** [comment]
Oh right, I think I just got confused because the article linked to "best in show one"

**Comment by BogdanCerovac** [comment]
Yes, sorry, the link pointed to nr. 1, yes, just to present all possibilities, but the code is from nr. 5.
